_posts/2019-02-25-the-western-grid.md

```diff
@@ -0,0 +1,136 @@
+</figure>
+<figure class="caption caption--right caption--flow">
+  <img src="{{ "/images/salt_lake_meridian.png" | prepend: site.baseurl }}" alt="Salt Lake Meridian" />
+  <figcaption class="caption__text">Fig 5: Salt Lake Meridian</figcaption>
+</figure>
+
+As with any measuring system, you need a starting point. Within the PLSS, this is called the Principal Meridian. In Utah, this location is in Salt Lake City at Main Street and South Temple (Fig 5). All official surveys in Utah have an underlying reference to this point. In other words, every Township and Section in Utah is either North, South, East, or West of this location. Other PLSS states have their own unique principal meridians (Fig 4).
+
+The Townships are made up of six square miles, with each square mile known as a Section. These Sections are eventually subdivided down to an acre by way of Half Sections, Quarter Sections, Quarter-Quarter Sections, etc. (Fig 6). An acre, again, is simply 10 square chains - and there are 640 acres in a Section (a square-mile) or 40 acres in a quarter-quarter section. Hence the phrase, “40 acres and mule” is all you needed to be self-sufficient.
+
+<figure class="caption caption--right">
+  <img src="{{ "/images/townships_sections.png" | prepend: site.baseurl }}" alt="Townships and Sections" />
+  <figcaption class="caption__text">Fig 6: Townships, Ranges and Sections</figcaption>
+</figure>
+
+No matter how you shake it, though, land measurements in the West are consistently divisible by the magic 66 foot Gunter Chain. You’ll see its effect everywhere. This measuring unit was so successful because it made the math easy - and many early surveyors had a shaky grasp of mathematics and they were required to work quickly.
+
+These early surveyors typically placed a permanent monument at Section and Quarter-Section corners. These markers are still heavily used today and are the starting point for almost every Western legal description you’ll encounter - including land deeds (the transfer of private property), water rights (well locations), taxing areas, municipal annexations - and consequently voting districts. Because of this, it has been difficult for the US to adopt the International System of Units (the modern metric system).
+
+It is precisely this survey pattern that explains why the Western United States is so orthogonal. Although the grid is not unique to the West, it's certainly at the core of all land sales and as a result it influences many development patterns. Let’s look at a few examples.
+
+## The Western Grid
+{: .text-left}
+
+<figure class="caption caption--left caption--flow">
+  <a href="{{ "/images/plat_of_zion.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/plat_of_zion_sm.png" | prepend: site.baseurl }}" class="width: 100%;" alt="Plat of Zion">
+  </a>
+  <figcaption class="caption__text">Fig 7: Plat of Zion</figcaption>
```

Comment:
Footnote 7: 
Frederick G. Williams, “Revised Plat of the City of Zion, Circa Early August 1833,” The Church of Jesus Christ of Latter-day Saints, The Joseph Smith Papers, updated September 01, 2018, accessed March 01, 2019. https://www.josephsmithpapers.org/paper-summary/revised-plat-of-the-city-of-zion-circa-early-august-1833/1.